src/app/components/tables/data-table/example/terra-data-table.interface.example.ts

```diff
@@ -0,0 +1,4 @@
+export interface DataTableExampleInterface
```

Comment:
```suggestion
export interface TerraDataTableInterfaceExample
```

analogous to the file name